ang/civicase/ContactCard.html

```diff
@@ -45,9 +45,9 @@
         <a
           title="{{ ts('View Contact') }}"
           href="{{ url('civicrm/contact/view', {cid: contact.contact_id}) }}">
-          <i class="civicase__contact-icon material-icons" ng-if="getContactIconOf(contact.contact_id) === 'Individual'">person</i>
-          <i class="civicase__contact-icon material-icons" ng-if="getContactIconOf(contact.contact_id) === 'Household'">home</i>
-          <i class="civicase__contact-icon material-icons" ng-if="getContactIconOf(contact.contact_id) === 'Organisation'">domain</i>
+          <i class="civicase__contact-icon-additional material-icons" ng-if="getContactIconOf(contact.contact_id) === 'Individual'">person</i>
```

Comment:
There is no mention of this class in the PR description. Can you please add a small bit of info about this?